app/js/views/services-list-item.jsx

```diff
@@ -8,10 +8,13 @@ export default class ServicesListItem extends React.Component {
       available: false,
       on: true,
       locked: true,
+      motionDetected: false,
```

Comment:
Again if this idea sticks, next step for us would be to finally split this view into several service specific mini-views. It's going to grow and become a total mess :)